CONTRIBUTING.md

```diff
@@ -230,20 +318,20 @@ tests/kuery/test_parser.py::ParserTests::test_number_wildcard_fail PASSED
 ```
 
 
-## Writing style
-
-Our rules are much more than queries. We capture a lot of metadata within the rules, such as severity, index pattterns, and noise level. We also have several fields that are user-readable text, such as `name`, `description`, `false_positives`, `investigation_notes`, and `name`. Those fields, which are populated with English text[*](#i18n-note), should follow the [Elastic UI writing guidelines](https://elastic.github.io/eui/#/guidelines/writing). We want our text to be *clear* and *concise*, *consistent* and *conversational*.
-
-<a id="i18n-note">\* Note</a>: We currently don't have i18n support for Detection Rules.
-
+### Uploading rules
 
-## Signing the contributor license agreement
+Once all of these steps are complete, you have the `upload-rule` can then upload your rule, whether that's part of your local infrastructure or the cloud.  
 
-Please make sure you've signed the [Contributor License Agreement](http://www.elastic.co/contributor-agreement/). We're not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
+1. This command demonstrates the syntax to upload to an [Elastic Cloud](#) instance:  
 
+```shell
+    python -m detection_rules kibana --kibana-url '<URL>' --kibana-user '<USER>' --kibana-password '<PASSWORD' upload-rule rules/<RULE_NAME>.toml
+```
```

Comment:
This is also in CLI.md

Reply:
yeah, I prefer to keep all CLI-specific examples in that file personally